packages/react/src/components/Stack/Stack.styles.ts

```diff
@@ -47,9 +47,11 @@ export const styles: IStackComponent['styles'] = (props, theme, tokens): IStackS
     textOverflow: 'ellipsis',
   };
 
+  const childSelector = enableScopedSelectors ? '.' + GlobalClassNames.child : '*';
```

Comment:
nit: technically `>` is part of the child selector, and always precedes `${childSelector}`. If you include it in the const:
```suggestion
  const childSelector = enableScopedSelectors ? '> .' + GlobalClassNames.child : '> *';
```

Then you can simplify below:
```diff
-          [`> ${childSelector}`]: {
+          [childSelector]: {
```

Reply:
Done!